iroh-relay/src/client/conn.rs

```diff
@@ -219,13 +227,15 @@ impl Sink<SendMessage> for Conn {
 
     fn poll_flush(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
         match *self {
+            #[cfg(not(wasm_browser))]
             Self::Relay { ref mut conn } => Pin::new(conn).poll_flush(cx).map_err(Into::into),
             Self::Ws { ref mut conn, .. } => Pin::new(conn).poll_flush(cx).map_err(Into::into),
         }
     }
 
     fn poll_close(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
         match *self {
+            #[cfg(not(wasm_browser))]
```

Comment:
just noting this down: it's really unfortunate we have to do this on all the branches, but I don't have a better idea

Reply:
Yeah I think we can remove them, once/if we go websocket connections only.